raidprotect/src/event/message/handle.rs

```diff
@@ -14,5 +21,41 @@ pub async fn handle_message(message: Message, state: Arc<ClusterState>) {
     let parsed = parse_message(&message);
     state.redis().set(&parsed).await.ok();
 
+    if is_an_old_command(&message) {
+        if let Err(error) = warn_about_old_command(&message, state.http()).await {
+            error!(error = ?error, "failed to warn user about question mark commands deprecation");
+        }
+    }
+
     info!("received message: {}", message.content) // Debug util real implementation
 }
+
+async fn warn_about_old_command(message: &Message, http: &Client) -> Result<()> {
+    let lang = Lang::fallback();
+    let embeds = [EmbedBuilder::new()
+        .title(lang.warning_deprecated_command_style_title())
+        .description(lang.warning_deprecated_command_style_description())
+        .color(COLOR_TRANSPARENT)
+        .build()];
+    http.create_message(message.channel_id)
+        .embeds(&embeds)?
+        .reply(message.id)
+        .exec()
+        .await
+        .context("warning about old commands with question  mark ?")?;
+    Ok(())
+}
+
+fn is_an_old_command(message: &Message) -> bool {
+    message.content.starts_with("?kick")
```

Comment:
To avoid duplication, we could store all the commands in a `const &[&str]` and use a `for` loop.